Language/Functions/Communication/Serial/readString.adoc

```diff
@@ -34,10 +34,49 @@ title: Serial.readString()
 === Returns
 A `String` read from the serial buffer
 
+
 --
 // OVERVIEW SECTION ENDS
 
 
+// HOW TO USE SECTION STARTS
+[#howtouse]
+--
+
+[float]
+=== Example Code
+Demonstrate Serial.readString()
+
+[source,arduino]
+----
+void setup() {
+  Serial.begin(9600);
+}
+
+void loop() {
+  Serial.println("Enter data:");
+  while(Serial.available()==0); //wait for data available
```

Comment:
```suggestion
  while (Serial.available() == 0) {}     //wait for data available
```

### Pad operators

This is the standard Arduino code formatting style, as implemented by the default configuration of the Arduino IDE's "**Auto Format**" feature

### Adjust `while` block style

I replaced the semicolon with braces in `while` block.

I think it is best practices to always wrap flow control blocks in braces, since omitting them makes the code less clear and is frequently the cause of confusing bugs.

I also think the formatting looks nicer with this approach compared to how it looks after the Arduino IDE 2.x "**Auto Format**" (which is a bit more comprehensive than Arduino IDE 1.x):

```cpp
  while (Serial.available() == 0)
    ;                                    //wait for data available
```